src/components/Auth/AuthMainForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import Switch from 'components/Shared/Switch';
+import StyledButton from 'components/Shared/Button';
+import PwdInput from 'components/Auth/PwdInput';
+import * as S from 'components/Auth/styles';
+
+const AuthForm = styled.div``;
+
+const AuthMainForm = () => (
+  <AuthForm>
+    <S.StyledInput autocomplete="username" name="username" placeholder="아이디 입력" />
+    <PwdInput placeholder="비밀번호 입력" />
+    <S.AutoLogin>
+      <S.AutoLoginCheck>
+        <Switch />
+      </S.AutoLoginCheck>
+      <S.AutoLoginText>자동 로그인</S.AutoLoginText>
+    </S.AutoLogin>
+    <StyledButton>로그인</StyledButton>
+    <S.OtherOption>
+      <S.StyledLink to="idfind">아이디 찾기</S.StyledLink>
+      <p> | </p>
+      <S.StyledLink to="pwdfind">비밀번호 찾기</S.StyledLink>
+      <div>|</div>
+      <S.StyledLink to="register">회원가입</S.StyledLink>
+    </S.OtherOption>
+    <S.SnsLoginText>SNS로 간편 로그인하기</S.SnsLoginText>
+    <S.OauthLogin>
+      <img src="/asset/kakaoLogo.webp" alt="kakao" />
+      <img src="/asset/naverLogo.webp" alt="kakao" />
+      <img src="/asset/googleLogo.webp" alt="kakao" />
+    </S.OauthLogin>
+    <S.NoneUser>비회원으로 이용하기</S.NoneUser>
+    <S.CopyRight>COPYRIGHT © 2021 BCSD LAB ALL RIGHTS RESERVED.</S.CopyRight>
```

Comment:
`2021` 은 변동성이 있는 문구입니다 ( 몇일후면 2022년이네요 :) )
다음과 같이 관리하는게 어떨까요?
```suggestion
    <S.CopyRight>COPYRIGHT © {new Date().getFullYear()} BCSD LAB ALL RIGHTS RESERVED.</S.CopyRight>
```

Reply:
오호 카피라이트 작성 방법 메모...
d28ea99